megamek/src/megamek/common/OffBoardDirection.java

```diff
@@ -78,4 +78,30 @@ public static OffBoardDirection getOpposite(OffBoardDirection value) {
             return NONE;
         }
     }
+    
+    /** 
+     * Returns the OffBoardDirection associated with given the on-board deployment
+     * position as defined in IStartingPositions.
+     */
+    public static OffBoardDirection translateStartPosition(int startPos) {
+        if (startPos > 10) {
```

Comment:
I'm having a little trouble understanding the logic in this method.

Reply:
I think it might mean modulo ten?

Reply:
The deep deployment zones use the numbers of the normal deployment zones +10